GithubRepo/Scenes/Search/SearchViewController.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  SearchViewController.swift
+//  GithubRepo
+//
+//  Created by Marcos Alves on 07/09/21.
+//
+
+import UIKit
+
+class SearchViewController: UIViewController {
+    
+    var searchTimer: Timer?
+    
+    private let kRepositoryTableViewCellIdentifier = String(describing: RepositoryTableViewCell.self)
```

Comment:
it will be better to place the cell identifier inside the cell class as a static property, as you did here